packages/mobile/src/pincode/PincodeSet.tsx

```diff
@@ -61,8 +62,11 @@ export class PincodeSet extends React.Component<Props, State> {
   }
 
   navigateToNextScreen = () => {
-    if (this.props.choseToRestoreAccount) {
+    if (this.props.route.params.changePin) {
+      navigate(Screens.Settings)
+    } else if (this.props.choseToRestoreAccount) {
       navigate(Screens.ImportWallet)
+      // if this prop is passed, then navigate back to Settings with a toast
```

Comment:
This comment looks out of place here and there's no toast being fired here as well

Reply:
thanks! this is one of the ones i wrote way earlier and forgot about. thanks!